CMakeLists.txt

```diff
@@ -58,6 +58,29 @@ elseif(CMAKE_SYSTEM_PROCESSOR_LC MATCHES "arm*")
 elseif(CMAKE_SYSTEM_PROCESSOR_LC MATCHES "ppc*" OR
   CMAKE_SYSTEM_PROCESSOR_LC MATCHES "powerpc*")
   set(CPU_TYPE powerpc)
+elseif(CMAKE_SYSTEM_PROCESSOR_LC MATCHES "mips*")
```

Comment:
Hmm. What are the processor names? `MATCHES` uses regexes, not globs, so this matches `mips`, `mip`, `mipss`, etc. Is `mips.*` intended?

The `ppc*` check above probably has the same thing.